store/config.go

```diff
@@ -64,6 +64,41 @@ func (c Config) CreateProductionLogger() *zap.Logger {
 	return logger.CreateProductionLogger(c.RootDir, c.LogLevel.Level)
 }
 
+func (c Config) String() string {
+	fmtConfig := "LOG_LEVEL: %v\n" +
+		"ROOT: %s\n" +
+		"PORT: %s\n" +
+		"USERNAME: %s\n" +
+		"PASSWORD: %s\n" +
```

Comment:
As a result of this ending up in output or logs, let's not print or record sensitive data like passwords. So let's just remove this line.